src/app/frontend/horizontalpodautoscalerlist/horizontalpodautoscalercard.html

```diff
@@ -0,0 +1,78 @@
+<!--
+Copyright 2015 Google Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<kd-resource-card object-meta="::$ctrl.horizontalPodAutoscaler.objectMeta" type-meta="::$ctrl.horizontalPodAutoscaler.typeMeta">
+  <kd-resource-card-columns>
+    <kd-resource-card-column>
+      <a ng-href="{{::$ctrl.getHorizontalPodAutoscalerDetailHref()}}">
+        <kd-middle-ellipsis display-string="{{::$ctrl.horizontalPodAutoscaler.objectMeta.name}}">
+        </kd-middle-ellipsis>
+      </a>
+    </kd-resource-card-column>
+    <kd-resource-card-column ng-if="$ctrl.areMultipleNamespacesSelected()">
+      <div>
+        <kd-middle-ellipsis display-string="{{::$ctrl.horizontalPodAutoscaler.objectMeta.namespace}}">
+        </kd-middle-ellipsis>
+      </div>
+    </kd-resource-card-column>
+    <kd-resource-card-column ng-if="$ctrl.showScaleTarget">
+      <a ng-href="{{::$ctrl.getScaleTargetHref()}}">
+        {{::$ctrl.horizontalPodAutoscaler.scaleTargetRef.kind}}: {{::$ctrl.horizontalPodAutoscaler.scaleTargetRef.name}}
+      </a>
+    </kd-resource-card-column>
+    <kd-resource-card-column>
+      <span ng-show="::$ctrl.horizontalPodAutoscaler.targetCPUUtilizationPercentage">
+      {{::$ctrl.horizontalPodAutoscaler.targetCPUUtilizationPercentage}}%
```

Comment:
Should the percent be localized here? Are there languages that percent comes first? Ask olek maybe :)